test/powershell/engine/ParameterBinding/ParameterBinding.Tests.ps1

```diff
@@ -281,6 +281,39 @@
         }
     }
 
+    It "Verify implicit argument conversion uses InvariantCulture" {
+        function Test-Function {
+            param([DateTime] $date)
+            $date
+        }
+
+        function Test-ScriptCmdlet {
+            [CmdletBinding()]
+            param([DateTime] $date)
+            $date
+        }
+
+        # Explict type cast use 'InvariantCulture' by default
+        $expected = [datetime] "1/11/1111"
```

Comment:
Why we think that is 'InvariantCulture' ? I think we should have explicit constructor and explicit check that "Explict type cast use 'InvariantCulture' by default". Also it is another test then "Verify implicit argument conversion uses InvariantCulture".

Reply:
Using invariant culture for a cast was a choice for consistency at the language level, casting a literal would always produce the same result in any culture.

If invariant culture conversions aren't appropriate, then one simply uses the appropriate code (constructor or whatever) and specifies how to do the conversion.

Reply:
@lzybkr Thanks for useful comment! But my question was just about the test - what if we break internally and replace InvariantCulture with other culture accidentally? So the line and comment will be wrong.

Reply:
Ah, I see, I was thinking it was a test for the conversion. I agree then, we should not rely on the default when checking results.

If we don't already, we should have a test that the cast is culture invariant.

Reply:
I will change this line to `LangaugePrimitives.ConvertTo` explicitly with `InvariantCulture`. And also will add a test to validate if the explicit conversion in powershell language is using 'InvariantCulture'.